src/neo/SmartContract/InteropService.Oracle.cs

```diff
@@ -0,0 +1,36 @@
+using Neo.Oracle.Protocols.Https;
+using System;
+
+namespace Neo.SmartContract
+{
+    partial class InteropService
+    {
+        public static class Oracle
+        {
+            public static readonly uint System_Oracle_Get = Register("System.Oracle.Get", Oracle_Get, 0, TriggerType.Application, CallFlags.None);
```

Comment:
Should this be renamed to `Neo.Oracle.Get`? I don't think that all blockchains need Oracle, which is unique to NEO.